src/factories/StandardChainlinkFactory.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.17;
+
+import {Ownable} from "openzeppelin-contracts/contracts/access/Ownable.sol";
+import {MutableUnlockersController} from "../controllers/MutableUnlockersController.sol";
+import {ChainlinkSourceAdapter} from "../adapters/source-adapters/ChainlinkSourceAdapter.sol";
+import {ChainlinkDestinationAdapter} from "../adapters/destination-adapters/ChainlinkDestinationAdapter.sol";
+import {IAggregatorV3Source} from "../interfaces/chainlink/IAggregatorV3Source.sol";
+import {BaseFactory} from "./BaseFactory.sol";
+
+/**
+ * @title OvalChainlink is the recommended Oval Chainlink contract that allows Oval to extract OEV generated by
+ * Chainlink usage.
+ */
+contract OvalChainlink is MutableUnlockersController, ChainlinkSourceAdapter, ChainlinkDestinationAdapter {
+    constructor(
+        IAggregatorV3Source source,
+        address[] memory unlockers,
+        uint256 lockWindow,
+        uint256 maxTraversal,
+        address owner
+    )
+        ChainlinkSourceAdapter(source)
+        MutableUnlockersController(lockWindow, maxTraversal, unlockers)
+        ChainlinkDestinationAdapter(18)
+    {
+        _transferOwnership(owner);
+    }
+}
+
+/**
+ * @title StandardChainlinkFactory is the recommended factory for use cases that want a Chainlink source and Chainlink
+ * interface.
+ * @dev This is the best factory for most use cases, but there are other variants that may be needed if different
+ * mutability choices are desired.
+ */
+contract StandardChainlinkFactory is Ownable, BaseFactory {
+    constructor(uint256 maxTraversal, address[] memory _defaultUnlockers)
+        BaseFactory(maxTraversal, _defaultUnlockers)
+    {}
+
+    /**
+     * @notice Creates the Chainlink Oval instance.
+     * @param source the Chainlink oracle source contract.
+     * @param lockWindow the lockWindow used for this Oval instance. This is the length of the window
+     * for the Oval auction to be run and, thus, the maximum time that prices will be delayed.
+     * @return oval deployed oval address.
+     */
+    function create(IAggregatorV3Source source, uint256 lockWindow) external returns (address oval) {
```

Comment:
Do you think there could be cases where someone wants to create an Oval instance with a different list of unlockers than the defaultUnlockers or max traversal?

Reply:
I think we want official Oval deployments to have the same unlocker setup, whereby default unlockers managed by Risk Labs are used.

I don't think these factories are intended for nonstandard deployments, but you can always do a custom deployment if you want that!